app/controllers/foo_controller.rb

```diff
@@ -1,6 +1,8 @@
 class FooController < ActionController::Base  
   def new_user
-    user = User.first(:conditions => ("username = '#{params[:username]}'"))
+    user = User.first(:conditions => "username = '#{params[:username]}'")
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Useless assignment to variable - `user`.](https://www.codacy.com/app/actcat/hogehoge/file/2965814407/issues/source?bid=3079895&fileBranchId=3428823#l3)